packages/connectivity/src/scp-cf/destination/destination-service-cache.ts

```diff
@@ -1,27 +1,25 @@
 import { JwtPayload } from '../jsonwebtoken-type';
-import { Cache, IsolationStrategy } from '../cache';
+import { Cache } from '../cache';
 import { Destination } from './destination-service-types';
-import { getDestinationCacheKey } from './destination-cache';
+import { getDestinationCacheKey, IsolationStrategy } from './destination-cache';
 
 const DestinationServiceCache = (cache: Cache<Destination[]>) => ({
   retrieveDestinationsFromCache: (
     targetUrl: string,
-    decodedJwt: JwtPayload,
-    isolationStrategy?: IsolationStrategy
+    decodedJwt: JwtPayload
   ): Destination[] | undefined =>
     cache.get(
-      getDestinationCacheKeyService(targetUrl, decodedJwt, isolationStrategy)
+      getDestinationCacheKey(decodedJwt, targetUrl, IsolationStrategy.Tenant)
```

Comment:
For this line and the line 20, I feel users lose some freedom for switching between tenant and tenant+user.
As the current know use cases do not need this flexibility, I'm happy to make it simple line now.